modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/SymbolMBLayerTest.java

```diff
@@ -36,4 +52,17 @@ public void testTextAnchorEnum(){
         assertEquals( 0.5, TextAnchor.getAnchorX("center"), 0.0 );
         assertEquals( 0.5, TextAnchor.getAnchorY("center"), 0.0 );
     }
-}
+
+    @Test
+    public void testTextMaxWidth(){
+        //Test default MBStyle value
+        assertEquals(10, testLayerDefault.getTextMaxWidth().intValue());
+        assertEquals(16, testLayerDefault.getTextSize().intValue());
+        assertEquals(160, testLayerDefault.getTextMaxWidth().intValue() * testLayerDefault.getTextSize().intValue());
```

Comment:
This line seems a bit redundant, but perhaps do a call to `testLayerDefault.transform()` and ensure that the correct value shows up in the resulting FeatureTypeStyle?